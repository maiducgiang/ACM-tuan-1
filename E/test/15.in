100
121813852 818901044
141036185 303580992
115254766 425354408
949423724 659140610
769753813 225162805
84060815 542709020
672546516 529666904
167895435 839263267
529318621 115529277
877499825 829228890
515512136 510443665
829675952 258546183
925296353 166483387
123540882 558854760
557723211 404592074
921326328 872265715
693114524 323107863
271874722 216629895
245200429 32707383
260727743 734492704
246465165 965711827
818362530 247286878
710011453 147832869
27009549 972274625
879974668 744398353
731030671 297609562
258366248 778546271
737639421 455168288
756034422 22991245
487834399 5136312
374415923 502078977
987218893 50666589
313601033 473806259
829540262 492632692
670287073 510209286
90920203 172089120
781189165 539435226
274875168 139191755
802701798 872541647
892881000 915210528
380939559 324212710
303119074 333694322
7503114 950356059
784510275 160690231
203361650 25383135
20541018 955234475
713752543 725392691
189190369 786466805
878170285 803325057
564141945 814399409
166465333 7283611
923199700 555419160
799690845 873375518
595899186 826467252
434923020 254560119
241666476 400761151
97923588 492835594
252768096 801516922
446793681 942417436
250537249 223622721
179089683 931604804
241899030 414634518
618715557 3437129
691153620 730683725
100740699 225555986
649936748 10855985
551998917 888017596
169883327 122137626
690680988 669472229
870240108 766128153
12642824 445964565
385212205 168840410
495617781 728135463
22699129 647834637
753426171 182774537
295913268 553320860
679080745 199894082
24096257 94951035
196504450 724770974
213307254 151568972
782213650 91324084
772853775 699028481
133899907 243266994
235713409 465755198
167537350 579873237
949125526 845272353
168610009 884156473
900615245 853340523
577480 359746802
948135799 381337091
470098772 419155537
310565259 14544868
260306280 57702286
757028044 203437509
359324677 703807245
268040759 157018019
124546752 7846352
901386249 67772471
295613462 244221107
295144873 738090957
