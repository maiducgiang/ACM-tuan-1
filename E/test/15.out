7
3
4
2
2
7
2
5
2
2
2
2
2
5
2
2
2
2
2
3
4
2
2
36
2
2
4
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
127
2
2
47
2
5
2
2
2
2
2
2
2
2
6
4
3
2
6
2
2
2
3
2
2
2
2
2
36
2
2
29
2
2
2
4
4
2
2
2
2
2
4
2
6
2
623
2
2
2
2
2
2
2
2
2
2
3
